src/qt3utils/datagenerators/daqsamplers.py

```diff
@@ -205,55 +202,51 @@ def __init__(self, daq_name='Dev1',
         self.read_write_timeout = read_write_timeout
         self.num_data_samples_per_batch = num_data_samples_per_batch
         self.trigger_terminal = trigger_terminal
+        self.sample_time = 1
```

Comment:
only meaningful change. all other changes were readded. Since this was just a parameter added to a class, I dont think this will break anything